pkg/runs/submitter.go

```diff
@@ -282,7 +282,7 @@ func (submitter *Submitter) submitRun(
 	readyRuns []TestRun,
 	submittedRuns map[string]*TestRun,
 	lostRuns map[string]*TestRun,
-	runOverrides *map[string]string,
+	runOverrides *map[string]string, // This doesn't appear to be used. Why not ?
```

Comment:
I think they are used by the jvmlauncher, if they are not then we have a pretty big bug

Reply:
No. It's not used. Does that mean overrides are just ignored ?

Reply:
Raised a separate issue to track this.